client/amnezia_application.cpp

```diff
@@ -163,6 +167,10 @@ void AmneziaApplication::init()
     m_pageController->showOnStartup();
 #endif
 
+#ifdef Q_OS_IOS
+    m_settingsController->toggleScreenshotsEnabled(m_settingsController->isScreenshotsEnabled());
```

Comment:
Is it possible to move this to the generic IOS initialization block above (line 121)?